components/Buttons/src/MDCButton.m

```diff
@@ -305,7 +310,22 @@ - (void)layoutSubviews {
 }
 
 - (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent *)event {
-  return CGRectContainsPoint(UIEdgeInsetsInsetRect(self.bounds, _hitAreaInsets), point);
+  // If there are custom hitAreaInsets, use those
+  if (!UIEdgeInsetsEqualToEdgeInsets(self.hitAreaInsets, UIEdgeInsetsZero)) {
+    return CGRectContainsPoint(UIEdgeInsetsInsetRect(self.bounds, self.hitAreaInsets), point);
```

Comment:
After writing the unit test, it seems that setting either the frame or the bounds property of a UIView standardizes the rect in the setter.  I will add the call to CGRectStandardize, but I'm having difficulty coming up with a unit test.